QUEST.tsv

```diff
@@ -2777,67 +2777,67 @@ QUEST_20151016_002784
+QUEST_20151022_002804	Take this strength.{nl}If this awful tower can be destroyed, I will do anything.
+QUEST_20151022_002805	By any chance, are you going to leave after receiving our strength?{nl}Since it's request by the General Carved Owl, I will lend my strength.
+QUEST_20151022_002806	My strength to be weak since it hasn't been long since I recovered my conscious.{nl}However, I hope that even this small amount of strength my help.
+QUEST_20151022_002807	Letter
+QUEST_20151022_002808	Even after running away from the demons that has invaded the castle, it'll be hard to survive. If anyone get hold of this letter, please pass it on to Yurus.. Eras
+QUEST_20151022_002809	Pillar is engraved with 1.
+QUEST_20151022_002810	Pillar is engraved with 2.
+QUEST_20151022_002811	Pillar is engraved with 3.
+QUEST_20151022_002812	Something is written faintly. A symbol.. A shard..?
+QUEST_20151022_002813	Thus, I decided to interfere.{nl}Even if it meant that I would be targetted by the demons.
+QUEST_20151022_002814	Area near us is the Kedoran Merchant Alliance's excavation site.{nl}Be cautious to cause no disturbance to the excavation.
+QUEST_20151022_002815	Detector is indicating it's close in the upper right corner.
+QUEST_20151022_002816	Detector is indicating it's close in the upper end.
+QUEST_20151022_002817	Detector is indicating it's close in the upper left corner.
+QUEST_20151022_002818	Detector is indicating it's a bit far away in the upper right corner.
+QUEST_20151022_002819	Detector is indicating it's a bit far away in the upper end.
+QUEST_20151022_002820	Detector is indicating it's a bit far away in the upper left corner.
+QUEST_20151022_002821	Detector is indicating it's far away in the upper right corner.
+QUEST_20151022_002822	Detector is indicating it's far away in the upper end.
+QUEST_20151022_002823	Detector is indicating it's far away in the upper left corner.
 QUEST_20151022_002824	
-QUEST_20151022_002825	
-QUEST_20151022_002826	
-QUEST_20151022_002827	
-QUEST_20151022_002828	
-QUEST_20151022_002829	
-QUEST_20151022_002830	
-QUEST_20151022_002831	
-QUEST_20151022_002832	
-QUEST_20151022_002833	
-QUEST_20151022_002834	
-QUEST_20151022_002835	
-QUEST_20151022_002836	
-QUEST_20151022_002837	
-QUEST_20151022_002838	
-QUEST_20151022_002839	
-QUEST_20151022_002840	
-QUEST_20151022_002841	
-QUEST_20151022_002842	
-QUEST_20151022_002843	
-QUEST_20151022_002844	
-QUEST_20151022_002845	
-QUEST_20151022_002846	
-QUEST_20151022_002847	
-QUEST_20151022_002848	
+QUEST_20151022_002825	Besides, those tombstones! It's suppose to release crimson energy.{nl}We were gathering holy energy to offer to the goddesses!{nl}
+QUEST_20151022_002826	Thank you for helping.{nl}We can take care of the rest.{nl}Let me thank you again.
+QUEST_20151022_002827	Requirement to create scripture is faith's
+QUEST_20151022_002828	They say what is the use in the world with disappeared goddesses.{nl}But we believe Goddesses will one day return and save us!{nl}
```

Comment:
There are some who doubt the use of trying when the goddesses have disappeared.{nl}However, we believe that the goddesses will return one day and save us!{nl}